index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1 class='header'>Hello World, we are software developers!</h1>
+    <section id="welcome-section" class="welcome-section">
+      <h1>Welcome Note</h1>
+        <p>This is to welcome you to our portfolio. Our names: Tinashe (Zimbabwe) and Frederick (Ghana). Be blessed!</p>
+      <a id="profile-link" href="https://github.com/Draxeytina/FrederickandTinashe.git" rel="stylesheet"></a></section>
+
+    </section>
+    <section id="projects">
+      <p class="project-tile">
+        <a href="https://github.com/Draxeytina/FrederickandTinashe.git" id="project-file">Check out our project</a>
+        <nav id="navbar" class="navbar">
+          <a href="fcc-test-suite-template-master/">Project</a>
+        </nav>
+        
```

Comment:
Try to avoid blank spaces especially within the same code block.

Reply:
- [ ]  Always make sure to include an explanation, a "why", for each requested change, and use polite words like kindly and It would be better. Kindly Implement it.

Reply:
Kindly try to avoid blank spaces so that your code is readable. You may apply this in the same code block.